docs/planning/backend/app/api/evaluations.py

```
@@ -0,0 +1,207 @@
+    db.commit()
+    return reviewer
+
+
+# --- Evaluation Routes ---
+
+@router.get("/evaluations", response_model=List[EvaluationResponse])
+def list_evaluations(
+    reviewer_id: int = Query(None, description="Filter by reviewer"),
+    fragrance_id: int = Query(None, description="Filter by fragrance"),
+    limit: int = Query(default=50, le=200),
+    db: Session = Depends(get_db)
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)